nav2_dwb_controller/dwb_plugins/src/simple_goal_checker.cpp

```diff
@@ -59,21 +63,35 @@ void SimpleGoalChecker::initialize(
     plugin_name + ".xy_goal_tolerance", rclcpp::ParameterValue(0.25));
   nav2_util::declare_parameter_if_not_declared(nh,
     plugin_name + ".yaw_goal_tolerance", rclcpp::ParameterValue(0.25));
+  nav2_util::declare_parameter_if_not_declared(nh,
```

Comment:
params / docs (and what does this one do?)

Reply:
This means if our class is stateful then the goal checker will not check if the xy position matches again once it is found to be true.

Reply:
Ah ok, its the eq. of latched in ROS1 nav, thanks.